src/common/utils.js

```diff
@@ -8,4 +8,36 @@ export const getVerificationStatus = (active, approved) => {
   } else if (active === false && approved === false) {
     return verificationStates.REJECTED;
   }
+};
+
+export const countAppliedFilters = (filter) => {
+  let numFilters = 0;
+
+  if (!filter) {
+    return numFilters;
+  }
+
+  if (filter['active'] !== undefined && filter['approved'] !== undefined) {
+    numFilters += 1;
+  }
+
+  Object.keys(filter).forEach(key => {
+    if (key === 'filter' || key == 'active' || key == 'approved') {
+      return;
+    } else if ((key === 'captureId' || key === 'uuid' || key === 'deviceIdentifier' || key === 'planterId' || key === 'planterIdentifier') && filter[key] !== '') {
+      numFilters += 1;
+    } else if (filter['tagId'] > 0) {
+      numFilters += 1;
+    } else if ((key === 'dateStart' || key === 'dateEnd') && filter[key] !== undefined) {
+      numFilters += 1;
+    } else if (key == 'organizationId' && filter[key] !== 'ALL_ORGANIZATIONS') {
```

Comment:
We have `ALL_ORGANIZATIONS` and `ALL_SPECIES` defined as variables in `src/models/filter.js`. It's best to use those in place of these string literals.